src/components/SubsidyRequestManagementTable/tests/CourseTitleCell.test.jsx

```diff
@@ -46,10 +46,9 @@ describe('CourseTitleCell', () => {
     renderWithRouter(Component);
     userEvent.click(screen.getByText(defaultProps.row.original.courseTitle));
     expect(screen.getByText('Loading course details...'));
-    waitFor(() => {
+    await waitFor(() => {
       screen.getByText(mockCourseDetails.shortDescription);
       screen.getByText('Learn more about this course');
-      screen.getByLabelText('in a new tab');
```

Comment:
Why remove this line? It's here to ensure this links is in fact opening in a new tab.

Reply:
Ah it's part of hyperlink. `getByText` will work here, not `getByLabelText`